package.json

```diff
@@ -1,34 +1,28 @@
 {
-  "name": "local-home-virual-environment",
+  "name": "local-home-testing",
   "version": "0.0.1",
-  "description": "",
+  "description": "Testing environment for Local Fulfillment Apps. Mocks behavior of Local Home Platform.",
+  "author": "Google",
   "private": true,
+  "main": "./src/index.ts",
```

Comment:
I believe the main usually points to the compiled Javascript location

Reply:
Changed to `./build/src/index.js`